### ISSUE TITLE
Problems with version reporting in CDAT82

### ISSUE THREAD

**Issue by durack1** [CDAT/cdat_info#35]
The function `cdat_info.get_version()` is reporting the wrong CDAT version - it should be `8.2`, but it's currently showing `8.1`:

```python
(base) bash-4.2$ conda activate cdat82MesaPy3
(cdat82MesaPy3) bash-4.2$ which python
~anaconda3/envs/cdat82MesaPy3/bin/python
(cdat82MesaPy3) bash-4.2$ python
Python 3.7.3 | packaged by conda-forge | (default, Jul  1 2019, 21:52:21) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import cdat_info
>>> print(cdat_info.get_version())
8.1
>>>
```

@downiec @muryanto1 @jasonb5

**Comment by muryanto1** [comment]
@durack1  cdat_info-8.2-py_6 has the fix. i.e. if you do: conda install -c conda-forge cdat_info, you should be getting the cdat_info that has correct version (8.2).
the 8.2 yaml files have not been regenerated though (as of 12/2).
I will regenerate at some point -- waiting to see if we want to include other fixes.

**Comment by doutriaux1** [comment]
@muryanto1 there's a way to generate the yamls so they do not include the build numbers (not sure if that's what we do at this point), it helps a lot and should allow you to not regenerate yamls in cases like this where only the build number changed.

**Comment by muryanto1** [comment]
> @muryanto1 there's a way to generate the yamls so they do not include the build numbers (not sure if that's what we do at this point), it helps a lot and should allow you to not regenerate yamls in cases like this where only the build number changed.

@doutriaux1 ow, actually our yaml files do not include build#. so I think yaml files are good, but I just need to update the latest package cdat_info-8.2-py_6 to cdat_forge, correct?

**Comment by doutriaux1** [comment]
Sounds right

**Comment by durack1** [comment]
@muryanto1 perfect thanks, what other fixes are you planning on implementing before minting the final 8.2 release?

**Comment by muryanto1** [comment]
@durack1  https://github.com/CDAT/vcs/issues/424 @forsyth2 is looking into this issue.

**Comment by jasonb5** [comment]
Closing as this was included in the 8.2.1 release.
